react/components/Payment/PaymentMethod.tsx

```diff
@@ -41,23 +56,20 @@ const PaymentMethod: FunctionComponent<Props & InjectedIntlProps> = ({
         <p className="c-on-base">{paymentGroupSwitch(payment, intl)}</p>
         {hasLastDigits && (
           <p className="c-muted-1 mb3">
-            {intl.formatMessage(
-              { id: 'payments.creditcard.lastDigits' },
-              {
+            <FormattedMessage
+              id="store/payments.creditcard.lastDigits"
+              values={{
                 lastDigits: payment.lastDigits,
-              }
-            )}
+              }}
+            />
           </p>
         )}
         <div className="flex items-center">
           <p className="c-muted-1 mv0">
             <Price value={payment.value} />
-            {` ${intl.formatMessage(
-              { id: 'payments.installments' },
-              {
-                installments: payment.installments,
-              }
-            )}`}
+            {` ${intl.formatMessage(messages.installments, {
```

Comment:
Why don't you use the `FormattedMessage` component here?

Reply:
Because it adds a space